docs-drafts/ci-cd.md

```diff
@@ -0,0 +1,67 @@
+# Commonly Used CI/CD Tools
+
+|  Tool |  Purpose | Support  | 
+|---|---|---|
+|  *Argo CD* |  GitOps continuous delivery tool for Kubernetes |  Availabe within OpenShift |
+|  *GitHub Actions*  |  GitOps continuous integration and delivery tool for Github |  Available for all GitHub repositories |
+|  *Tekton*  | GitOps continuous integration and delivery tool |  Available within OpenShift |
+
+## Argo CD
+
+Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes (the foundation of OpenShift). You can use it to implement a GitOps-style deployment service. 
+
+### Why Argo CD
+
+ArgoCD provides more control over the success of the deployment and brings many other benefits of the infrastructure as code/GitOps approach, such as:
+
+* Observability
+
+* Improved stability and consistency
+
+* An improved security model
+
+* It is efficient, well supported, and well documented.
+
+* It can reduce the maintenance overhead of pipelines
+
+* It will help you to reduce resource consumption by using a shared service
```

Comment:
Can we say, "It helps you to reduce..." > changing to active voice without changing meaning? The previous line meaning would change if it was active voice, not so sure about this one.

Reply:
For sure, no concerns with that change.